README.md

```diff
@@ -13,81 +13,25 @@ displays the contacts themselves, fetching them from content store:
 
 ## Technical documentation
 
-This is a Ruby on Rails application that provides a private admin UI for signon users with permission to allow them to manage the contacts for their organisation.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
```

Comment:
Commit 36dda45fc8d13cbf2256eb6bc80c91856d00eefa - the commit message references link `[1]` but doesn't provide a link 🤔

Reply:
Oops, thanks for spotting. Fixed.